dashboard/src/actions/charts.ts

```diff
@@ -119,29 +115,13 @@ export function fetchCharts(
           nextPageToken: response.nextPageToken,
         }),
       );
+      dispatch(receiveChartCategories(response.categories));
```

Comment:
This is the key change, once we don't consider the FilterOptions (https://github.com/kubeapps/kubeapps/pull/3225) we can safely rely on the same fetchCharts calls to update the categories.